articles/tokens/legacy/refresh-token.md

```diff
@@ -0,0 +1,153 @@
+---
+description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+---
+
```

Comment:
@cabralmartinThis is the legacy doc. I removed some of the content that I suppose we don't want there (for example referring to using id_tokens to invoke an API). I left the obtain/use/revoke paragraphs as they were. Let me know what you think.